packages/@aws-cdk/aws-cloudtrail/README.md

```diff
@@ -92,4 +92,33 @@ trail.addLambdaEventSelector(["arn:aws:lambda"]);
 
 // Add an event selector to log data events for the provided Lambda functions.
 trail.addLambdaEventSelector([lambdaFunction.functionArn]);
+```
+
+Using the `Trail.onEvent()` API, a Cloudwatch event rule can be created that gets triggered for
```

Comment:
```suggestion
Using the `Trail.onEvent()` API, an EventBridge rule can be created that gets triggered for
```

might be for a different PR
motivation: consistency with upcoming change in #8020